// ag-charts-angular/esm2022/lib/ag-gauge.component.mjs
import { Component, EventEmitter, Input, Output, ViewEncapsulation } from '@angular/core';
import { AgCharts as AgChartsAPI } from 'ag-charts-community';
import { AgChartsBase } from './ag-charts-base';
import * as i0 from "@angular/core";
// noinspection AngularIncorrectTemplateDefinition
export class AgGauge extends AgChartsBase {
    ngZone;
    options = { type: 'radial-gauge', value: 0 };
    onChartReady = new EventEmitter();
    constructor(elementDef, ngZone) {
        super();
        this.ngZone = ngZone;
        this._nativeElement = elementDef.nativeElement;
    }
    createChart(options) {
        return AgChartsAPI.createGauge(options);
    }
    static ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.2.12", ngImport: i0, type: AgGauge, deps: [{ token: i0.ElementRef }, { token: i0.NgZone }], target: i0.ɵɵFactoryTarget.Component });
    static ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "14.0.0", version: "16.2.12", type: AgGauge, isStandalone: true, selector: "ag-gauge", inputs: { options: "options" }, outputs: { onChartReady: "onChartReady" }, usesInheritance: true, ngImport: i0, template: '', isInline: true, encapsulation: i0.ViewEncapsulation.None });
}
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.2.12", ngImport: i0, type: AgGauge, decorators: [{
            type: Component,
            args: [{
                    selector: 'ag-gauge',
                    standalone: true,
                    template: '',
                    encapsulation: ViewEncapsulation.None,
                }]
        }], ctorParameters: function () { return [{ type: i0.ElementRef }, { type: i0.NgZone }]; }, propDecorators: { options: [{
                type: Input
            }], onChartReady: [{
                type: Output
            }] } });
